### PR TITLE
VB-3535 Retrieve available visit sessions

### PR DESCRIPTION
**Pull request by tpmcgowan**
* Add basic client and service layer implementations to get available visit sessions for a prison / prisoner / visitors selection
* So far just dump out the retrieved data

### CODE REVIEW

File: server/routes/bookingJourney/selectDateAndTimeController.ts [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
-import { BookerService } from '../../services'
+import { VisitSessionsService } from '../../services'
 
 export default class DateAndTimeController {
-  public constructor(private readonly bookerService: BookerService) {}
+  public constructor(private readonly visitSessionsService: VisitSessionsService) {}
 
   public view(): RequestHandler {
     return async (req, res) => {
+      const { prisoner, selectedVisitors } = req.session.bookingJourney
```
Comment (tpmcgowan) [inline server/routes/bookingJourney/selectDateAndTimeController.ts:9]: This route and template so far just dump out the retrieved data

File: server/views/pages/bookingJourney/selectDateAndTime.njk [hunk 2]
```diff
@@ -7,4 +7,6 @@
   BOOKING JOURNEY
   {{ bookingJourney | dump(2) }}
 
+  AVAILABLE VISIT SESSIONS
```
Comment (tpmcgowan) [inline server/views/pages/bookingJourney/selectDateAndTime.njk:10]: Next PR will replace this dummy page with the actual calendar 📆 !

File: server/views/pages/bookingJourney/selectDateAndTime.njk [hunk 3]
```diff
@@ -7,4 +7,6 @@
...
+  {{ visitSessions | dump(2) }}
 </pre>
```
Comment (psoleckimoj) [inline server/views/pages/bookingJourney/selectDateAndTime.njk:12]: Trailing newline 😛

Comment (tpmcgowan) [reply server/views/pages/bookingJourney/selectDateAndTime.njk:12]: Thought you might pick up on that when I saw the red marker 😂 ! Don't worry: it'll be gone soon...

**Pull request merged**